src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * It records a transaction of a tag addition/deletion
+ */
+public class Tagging {
+    private Person person;
+    private Tag tag;
+    private boolean addition; // true means it is an addition, false means deletion
```

Comment:
Coding standard violation for boolean.
Using boolean is fine now since this LO requires only two states. What if later you need to capture more than two states?